docker/docker-compose.yml

```diff
@@ -15,6 +15,8 @@ services:
     build:
       context: ../src/api/status
       dockerfile: Dockerfile
+      cache_from:
+        - docker.cdot.systems/status:buildcache
```

Comment:
Awesome to see us using build cache now!

Question: since this service doesn't exist in the Docker Registry yet, docker knows to skip this cache_from step when building, right?

Reply:
I *think* what will happen is that it will just treat this as the cache not existing and build normally.  We'll have to test.

Reply:
First time seeing `cache_from` in `docker-compose`, so if I understand correctly it will check for layers that can use cache from our registry, and if there is no cache built from our context, right?

Do we also need to do the same thing for feed_discovery?

Reply:
Yes, you are correct: it will download vs. re-create layers that already exist in the registry.

NOTE: I already do the `feed-discovery` service here https://github.com/Seneca-CDOT/telescope/pull/3207/files#diff-b020a27fee741b5e10a4d106e6d266ec2bc9a4e694496470529407966142ad35R101